src/parpeoptimization/optimizationOptions.cpp

```diff
@@ -308,7 +308,7 @@ Optimizer* optimizerFactory(optimizerName optimizer)
 }
 
 
-void printAvailableOptimizers()
+void printAvailableOptimizers(std::string prefix)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function parameter 'prefix' should be passed by const reference.](https://app.codacy.com/manual/dweindl/parPE/pullRequest?prid=4904108)